src/common/components/header/index.js

```diff
@@ -2,73 +2,104 @@ import React, { Component, PropTypes } from 'react';
+          <a href="#navigation-closed" className={ style['p-navigation__toggle--close'] } title="close menu">Close menu</a>
         </div>
+        <nav className={ style['p-navigation__nav'] } role="menubar">
+          <ul className={ style['p-navigation__links']} role="menu">
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a href={ `${SNAPCRAFT_URL}/store` }>Store</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a href={ `${SNAPCRAFT_URL}/blog` }>Blog</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a className={ authenticated ? '' : style['is-selected'] } href={ `${SNAPCRAFT_URL}/build}` }>Build</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a href="https://docs.snapcraft.io">Docs</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a className={ style['p-link--external'] } href="https://forum.snapcraft.io/categories">Forum</a>
+            </li>
+          </ul>
+          { authenticated
+            ?
+              <ul className={ style['p-navigation__links--right']} role="menu">
+                <li className={ style['p-navigation__link']} role="menuitem">
+                  <a className={ style['p-dropdown__toggle']} aria-controls="account-menu" aria-expanded={ this.state.showUserDropdown }
+                    onClick={ this.onDropdownClick.bind(this)}
+                  >
+                    {user.name || user.login}<IconChevron/>
+                  </a>
+                  <ul className={ style['p-dropdown__menu']} id="account-menu" aria-hidden={ !this.state.showUserDropdown }>
+                    <li className={ style['p-navigation__link']} role="menuitem">
+                      <a href={ `${SNAPCRAFT_URL}/account/snaps` } >My published snaps</a>
+                    </li>
+                    <li className={ style['p-navigation__link']} role="menuitem">
+                      <a href={ `/user/${user.login}`} className={ style['is-selected']}>Build with GitHub</a>
+                    </li>
+                    <li className={ style['p-navigation__link']} role="menuitem">
+                      <a href={ `${SNAPCRAFT_URL}/account/details` }>Account details</a>
+                    </li>
+                    <li className={ style['p-navigation__link']} role="menuitem">
+                      <a href="/auth/logout" onClick={ this.onLogoutClick.bind(this)}>Sign out</a>
+                    </li>
+                  </ul>
+                </li>
+              </ul>
+            :
+              <ul className={ style['p-navigation__links--right']} role="menu">
+                <li className={ style['p-navigation__link'] } role="menuitem">
+                  <a href={ `${SNAPCRAFT_URL}/login?next=/login/bsi` }>
```

Comment:
Correct. Part of the reason it has 'Don't merge' is so that we can run it by people and gauge thoughts. In the future when we eventually move BSI into SI this will also be the case!

Reply:
Also, now that BSI login goes through snapcraft.io login we can't deploy BSI before snapcraft.io, because it will be impossible to log in to BSI before snapcraft.io goes live. Especially that it seems that `/login/bsi` part on snapcraft.io side was made in the same PR as rest of logout redirect code that was meant to be safely deployed after.

I know that if we coordinate it well we could deploy snapcraft.io quickly enough to minimise possibility of this issue affecting anyone, but in case of any deployment issues we will end up with BSI deployment broken until snapcraft.io is deployed, which we should avoid if possible.

If we want to keep BSI login going through snapcraft.io/login - snapcraft.io login code should be deployed first as a small update, followed by BSI, followed by snapcraft.io logout deployment. So that at any point both BSI and snapcraft.io would be fully functional.

Reply:
This has now been removed :)